NEWS.md

```diff
@@ -1,5 +1,8 @@
 ## 20.1.0 In-progress
 
+* Set a default value of false for instance.previouslyHeld (https://issues.folio.org/browse/MODINVSTOR-454)
```

Comment:
If we include links to JIRA issues, I think it is better to provide a label for them that shortens the length of the line in the formatted markdown.

```suggestion
* Defaults instance.previouslyHeld to false ([MODINVSTOR-454](https://issues.folio.org/browse/MODINVSTOR-454))
```